sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClientBuilder.java

```diff
@@ -69,27 +71,27 @@
  */
 @ServiceClientBuilder(serviceClients = {TextAnalyticsAsyncClient.class, TextAnalyticsClient.class})
 public final class TextAnalyticsClientBuilder {
+    private static final String DEFAULT_SCOPE = "https://cognitiveservices.azure.com/.default";
+    private static final String NAME = "name";
     private static final String OCP_APIM_SUBSCRIPTION_KEY = "Ocp-Apim-Subscription-Key";
-    private static final String ECHO_REQUEST_ID_HEADER = "x-ms-return-client-request-id";
```

Comment:
Hi @johanste Shawn noticed that we don't have a way to tell the Text Analytics cognitive service to echo "x-ms-client-request-id" in the response, i.e., service won't honor "x-ms-return-client-request-id", which means the caller cannot correlate the request with a response.

We want to check with you that this is something we need to communicate to the service team?  Is this something we expect the service to follow, or is it optional?

[Shawn is also checking behavior of Form Recognizer meantime]